model/license.go

```diff
@@ -64,6 +64,7 @@ type Features struct {
 	GuestAccountsPermissions  *bool `json:"guest_accounts_permissions"`
 	IDLoadedPushNotifications *bool `json:"id_loaded"`
 	LockTeammateNameDisplay   *bool `json:"lock_teammate_name_display"`
+	EnterprisePlugins         *bool `json:"enterprise_plugins"`
```

Comment:
Not going to hold up the review for this, but I wonder if we shouldn't just send up the `future_features` flag. Adding a field here makes sense for  changes /in/ the server we might license discretely in the future, but this bucket is just way too wide to be useful for licensing enterprise plugins. Since we don't have a packaging story, should we just key off the existing "default" bit?

Reply:
Adding a new feature flag has almost now cost. Adding a flag right now given us a bit flexibility if there is the decision to package enterprise plugins into a new SKU. Also, if this flag is considered obsolete in the future, we can just drop it without compatibility issues.